### PR TITLE
fix: confluxJS.provider

### PR DESCRIPTION
**Pull request by yqrashawn**
PORTAL-1754

---
This change is [Reviewable](https://reviewable.io/reviews/conflux-chain/helios/362)

### CODE REVIEW

**Comment by angelia-yuqi-personal** [comment]
Task linked: [PORTAL-1754 fluent不支持scan中的FC-CFX Non-reversible Exchange](https://app.clickup.com/t/3719212/PORTAL-1754)

**Comment by angelia-yuqi-personal** [comment]
Task linked: [PORTAL-1589 dapp approve 无法解析token 字段是空。](https://app.clickup.com/t/3719212/PORTAL-1589)

**Review by angelia-yuqi-personal — DISMISSED** [review]

**Comment by angelia-yuqi-personal** [comment]
Task linked: [PORTAL-1798 添加FC后，获取不到FC的balance](https://app.clickup.com/t/3719212/PORTAL-1798)

**Comment by angelia-yuqi-personal** [comment]
Task linked: [PORTAL-1876 修复 Bug: 添加FC后，获取不到FC的balance](https://app.clickup.com/t/3719212/PORTAL-1876)

**Comment by angelia-yuqi-personal** [comment]
Task linked: [CU-1zpjtq7 integrate sentry](https://app.clickup.com/t/1zpjtq7)

**Comment by angelia-yuqi-personal** [comment]
Task linked: [PORTAL-1892 pospool进行Stake操作，ledger会先接收到transcation，进行签名后，fluent才弹出来，然后再进行签名，Stack操作才会完成](https://app.clickup.com/t/3719212/PORTAL-1892)

**Comment by angelia-yuqi-personal** [comment]
Task linked: [PORTAL-1587 交易列表数据原来是12条。突然变成了2条。不稳定复现](https://app.clickup.com/t/3719212/PORTAL-1587)

**Comment by angelia-yuqi-personal** [comment]
Task linked: [PORTAL-1692 交易列表有些成功的历史交易消失不见了](https://app.clickup.com/t/3719212/PORTAL-1692)

**Comment by angelia-yuqi-personal** [comment]
Task linked: [PORTAL-1774 PandaID 还没连接Fluent ,"Connect Wallet"按钮当前就显示成“Personal Sign” 对比portal，未连接时按钮显示“Connect Wallet” 连接成功后才显示“Personal Sign”](https://app.clickup.com/t/3719212/PORTAL-1774)

**Comment by angelia-yuqi-personal** [comment]
Task linked: [PORTAL-1725 发送token MAX资产成功后，该账号首页资产列表对应token余额不更新](https://app.clickup.com/t/3719212/PORTAL-1725)

File: packages/rpcs/cfx_signTransaction/index.js [hunk 1]
```diff
@@ -131,15 +132,25 @@ export const main = async args => {
...
+        addressId: fromAddr.eid,
+        device: fromAddr.account.accountGroup.vault.device,
+      })
+    }
   } else {
-    const pk = await wallet_getAddressPrivateKey({address: newTx.from})
+    let pk = await wallet_getAddressPrivateKey({address: newTx.from})
 
+    if (dryRun)
+      pk = '0x617a260992af90f20fbbe43cd2e2a4edb9a0ec2ded468de133797184be3cd818'
```
Comment (zctocm) [inline packages/rpcs/cfx_signTransaction/index.js:153]: this pk is for testing?

Comment (yqrashawn) [reply packages/rpcs/cfx_signTransaction/index.js:153]: It's for dry run only. 
Sign everything with this pk when dry run to ensure the tx is singable

**Review by zctocm — APPROVED** [review]

**Pull request merged**